generator/src/main.ts

```diff
@@ -101,6 +101,10 @@ function main(): void {
             'src/extension.ts'
         )
 
+        shell.exec(
+            `bash -c 'cat package.json | jq --argjson icon "$(echo -n "data:image/png;base64,$(base64 < ../icons/${languageID}.png)" | jq -R -s .)" ".icon = \\$icon" > package.json'`
+        )
```

Comment:
To me it seems like it would be a lot easier (esp. to understand) to just use `fs.readFile()` + `JSON.parse()` + `new Buffer().toString('base64')` + `JSON.stringify()`.

I personally have no idea how `jq` works, what the flags here mean etc.

Reply:
Switched from `jq` to Node functions